cmd/podman/root.go

```diff
@@ -191,8 +191,9 @@ func loggingHook() {
 
 func rootFlags(opts *entities.PodmanConfig, flags *pflag.FlagSet) {
 	// V2 flags
-	flags.StringVarP(&opts.Uri, "remote", "r", registry.DefaultAPIAddress(), "URL to access Podman service")
-	flags.StringSliceVar(&opts.Identities, "identity", []string{}, "path to SSH identity file")
+	flags.StringVarP(&opts.Uri, "remote", "r", registry.DefaultAPIAddress(), "URL to access Podman servic (PODMAN_HOST)")
+	flags.StringSliceVar(&opts.Identities, "identity", []string{}, "path to SSH identity file, (PODMAN_SSHKEY)")
+	flags.StringVar(&opts.PassPhrase, "passphrase", "", "passphrase for identity file (not secure, PODMAN_PASSPHRASE), ssh-agent always supported")
```

Comment:
I really can't think of any situation in which this would be desirable.

Reply:
I would call it password  And have it support "-"  to take password from stdin.

podman login --password, -p is already supported.

Reply:
@rhatdan I am trying to differentiate between the password for authentication to the remote server vs the passphrase to open the key file.

@edsantiago It is up to the user to determine when it's acceptable to give the phrase on the command line.